seed/seed20_test.go

```diff
@@ -89,7 +90,11 @@ func (h *testSnapHandler) HandleAndDigestAssertedSnap(name, path string, essType
 		}
 		h.asserted[name] = fmt.Sprintf("%s:%s:%s", h.rel(path), essType, revno)
 	}()
-	return snapSHA3_384, sz, err
+	if essType != "gadget" {
+		// XXX seed logic actually reads the gadget, leave it alone
+		path = h.pathPrefix + path
```

Comment:
Is this about seed test logic only?

Reply:
if we do this for gadget the tests fail, because seed does ReadInfo  on it using the path,  we could have this code make a real copy somewhere but it feels overkill/slower, but the price is then this awkward detail. It's a test cheating too much, the code itself is fine.

Reply:
Thanks, that was my only concern, it's fine then.